template/web/wp-content/mu-plugins/autoload.php

```diff
@@ -0,0 +1,13 @@
+<?php
```

Comment:
The [boxuk/wp-muplugin-loader](https://github.com/boxuk/wp-muplugin-loader) package handles the autoloading of MU plugins in directories, this file is not needed.